source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -0,0 +1,264 @@
+
+      Figure 11: |CL| installer partitions defined
+
+12. Complete the remaining steps of :guilabel:`Required Options` to 
+    to install |CL|. Complete any :guilabel:`Advanced Options` as desired.
+    
+
+Ways to boot |CL|
+*****************
+
+Although we installed |CL| last, Ubuntu is still the default boot OS.  There are several ways to boot |CL|:
+
+#. Use your BIOS “Boot Menu” to select and boot |CL|.  
+   This is temporary and will not make |CL| the default boot OS.  
+
+#. Make systemd-boot, the boot loader that |CL| uses, the default
+   boot loader to boot |CL| and chain-boot GRUB, therefore boot 
+   Ubuntu. Follow the next section to implement this.  
+
+#. Make systemd-boot the Default Bootloader and Chain-boot GRUB.
+
+   * Boot up the |CL| installer image.
+   * Open a terminal window. 
+   * Identify the EFI system partition, Ubuntu root partition, and |CL| root
+     partition. 
+
+     .. code-block:: bash
+
+        sudo fdisk -l
+
+     Example output:
+
+
+     .. code-block:: console
+
+        clrlinux@clr-live~ $ sudo fdisk -l 
+        ... 
+
+        Disk /dev/sda: 335.4 GiB, 360080695296 bytes, 703282608 sectors
+        Disk model: INTEL SSDSCKKF36
+        Units: sectors of 1 * 512 = 512 bytes
+        Sector size (logical/physical): 512 bytes / 512 bytes
+        I/O size (minimum/optimal): 512 bytes / 1048576 bytes
+        Disklabel type: gpt
+        Disk identifier: D5CB69E9-2C27-4A16-9552-3CD6BFA5DA77
+
+        Device         Start       End   Sectors   Size Type
+        /dev/sda1       2048   1050623   1048576   512M EFI System
+        /dev/sda2    1050624 498481151 497430528 237.2G Linux filesystem
+        /dev/sda3  498481152 498993151    512000   250M Linux swap
+        /dev/sda4  498993152 703281151 204288000  97.4G Linux root (x86-64)
+        ...
+
+The above example output contains these partitions:
+
+* /dev/sda1 is the EFI system partition originally created by Ubuntu 
+  and shared with |CL|
+* /dev/sda2 is the Ubuntu root partition
+* /dev/sda3 is the swap partition for |CL|
+* /dev/sda4 is the |CL| root partition
+
+The remaining steps will work with these partitions.
+
+#. Mount these partitions
+
+   .. code-block:: bash
+      
+      sudo mkdir /mnt/clearlinux
+      sudo mount /dev/sda4 /mnt/clearlinux/
+      sudo mount /dev/sda1 /mnt/clearlinux/boot
+
+#. Make systemd-boot the default bootloader
+
+   .. code-block:: bash	
+
+      sudo bootctl install --esp-path=/mnt/clearlinux/boot
+
+#. Add a timeout to systemd-boot so that it will present the menu of
+   bootable OSes and give you time to select the one you want to boot. 
+
+   .. code-block:: bash	
+
+      sudo clr-boot-manager set-timeout 20 --path=/mnt/clearlinux
+      sudo clr-boot-manager update --path=/mnt/clearlinux
+#. Add a system-boot boot entry for GRUB
+   
+   .. code-block:: bash	
+
+      sudo tee -a /mnt/clearlinux/boot/loader/entries/grub.conf << EOF
+      title Ubuntu 19.04 Desktop
```

Comment:
@bktan8 I revised the `title` from "GRUB Menu" to "Ubuntu 19.04 Desktop." 
This seems more logical because it defines what's accessible in the GRUB, versus the GRUB menu itself.

Reply:
This should stay as "GRUB Menu" because it could be other operating systems listed in the GRUB menu.  I know this one particular example is simply Ubuntu.  But I want to make it more general.